bxin/ptg2m1m3/mountTelGenerator.ipynb

```diff
@@ -144,13 +174,52 @@
    "metadata": {},
    "outputs": [],
    "source": [
+    "#for setting constant elevation and azimuth angles.\n",
     "start_time = Time(datetime.now())\n",
     "freq = 1 #Hz\n",
     "end_time = start_time + timedelta(minutes=80)\n",
     "while Time(datetime.now())< end_time:\n",
     "    await asyncio.sleep(1.0/freq)\n",
     "    #mountCtrl.tel_elevation.set_put(angleActual=mountAngle.angleActual+dAngle)\n",
-    "    mountCtrl.tel_elevation.set_put(angleActual=89.5)"
+    "    #mountCtrl.tel_elevation.set_put(angleActual=89.5)\n",
+    "    #mountCtrl.tel_elevation.set_put(actualPosition=75)\n",
+    "    mountCtrl.tel_elevation.set_put(actualPosition=88)\n",
+    "    mountCtrl.tel_azimuth.set_put(actualPosition=45.0)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "#this cell is used when we do not know how much we can deviate \n",
+    "#from the real elevation angle (as measured by the inclinometer)\n",
+    "# we change the angle as a function of time, and stop at round numbers\n",
+    "\n",
+    "freq = 1 #Hz\n",
+    "startAngle = 86\n",
+    "stopAngle = 75\n",
+    "vAngle = 1 #1 deg change per minute\n",
+    "holdMinutes = 2\n",
+    "angleStepSize = 1 #each time we change by 1 deg, before we hold in place\n",
+    "\n",
+    "rampMinutes = angleStepSize/vAngle\n",
+    "print('This will run for %.0f minutes'%((startAngle - stopAngle)*(rampMinutes+holdMinutes)))\n",
+    "start_time = Time(datetime.now())\n",
+    "startTime = time.time()\n",
+    "end_time = start_time + timedelta(minutes=80)\n",
+    "demandAngle = startAngle\n",
+    "while demandAngle > stopAngle-0.01:\n",
+    "    await asyncio.sleep(1.0/freq)\n",
+    "    \n",
+    "    timeNow = time.time()\n",
+    "    minutesEllapsed = (timeNow - startTime)/60\n",
+    "    cyclePassed = np.floor(minutesEllapsed/(rampMinutes+holdMinutes))\n",
+    "    minutesIntoThisCycle = min(1, minutesEllapsed - cyclePassed*(1+holdMinutes))\n",
+    "    demandAngle = startAngle -  cyclePassed*1 - minutesIntoThisCycle * vAngle\n",
```

Comment:
Why `cyclePassed*1` instead of `cyclePassed`?

Reply:
Good catch. Fixed a few other hardcoded values along with this.